Source/Core/InputCommon/InputConfig.h

```diff
@@ -34,7 +34,8 @@ class InputConfig
     m_controllers.emplace_back(std::make_unique<T>(std::forward<Args>(args)...));
   }
 
-  ControllerEmu::EmulatedController* GetController(int index);
+  ControllerEmu::EmulatedController* GetController(int index) const;
+  int GetControllersNum() const;
```

Comment:
This seems unused, did you mean to put this in another PR instead?

Reply:
No I specified I will use in another PR in the commit (input caching), it was next to another change so I just thought I'd push this as well. Not too fuzzed about it, but it could have other uses as well.

Reply:
No preference either way, it just raises an eyebrow here when you add a method but never use it. Makes it appear as incomplete change (which either misses the call, or was meant for something else).

Reply:
I can take it out if it's a problem, but I'm fairly sure the rest of the input caching will be merged, PR is coming soon.